### DISCUSSION TITLE
Resources for Docker Image

### DISCUSSION THREAD

**Discussion by mbtools** [gitpod-io/openvscode-server#106] [Q&A] [answered]
How much CPU, memory, and disk space do I need to run the Docker image?
https://hub.docker.com/r/gitpod/openvscode-server

**Comment by akosyakov** [comment, marked as answer]
Server does not consume much if you do nothin. How much really depends on which tools and languages you will use and size of your projects. If you open this repo it can eat like 2G of RAM sometimes, but it because typescript compiler has to process so much code. There is no general answer, sorry.
